packages/core-assortments/db/helpers.js

```diff
@@ -110,6 +110,32 @@ Collections.Assortments.findAssortments = ({
   return Collections.Assortments.find(selector, options).fetch();
 };
 
+Collections.Assortments.count = async ({
```

Comment:
Super nice you have added these count functions to the collections but please extract the selector building code to minimize duplication and reduce likelihood of wrong counts when just one selector got changed in one function.

Kind of like:
```
const buildFindSelector = (props) => ({ isActive: ..., ... }); // contains selector building code of findAssortments
Assortments.findAssortments = (props) { sort=..., Assortments.find(buildFindSelector(props), sort).fetch() }
Assortments.count = (props) { Assortments.find(buildFindSelector(props)).count() }
```

Reply:
@pozylon while i agree making the field names uniform is good i think some of the fields that you outlined above are descriptive and shouldn't be deprecated like  `filteredProducts` while `examinedProducts` can be replaced with `productsCount`

Reply:
Agreed, makes sense